sdk/samples/iot/aziot_esp8266/aziot_esp8266.ino

```diff
@@ -268,6 +268,19 @@ void setup()
   establishConnection();
 }
 
+static uint8_t m_payload[100];
```

Comment:
What does m_ mean?

Reply:
Old syntax for "member". I was going to rename it anyway, just waiting for Wellington to verify the change against Time Series Insights.